packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -967,6 +967,24 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             metadata,
         );
 
+        const shouldRunExpiry = this.runtimeOptions.gcOptions?.gcAllowed === true
+        && this.runtimeOptions.gcOptions?.disableGC !== true
+        && this.runtimeOptions.gcOptions?.runSweep === true;
+
+        if(shouldRunExpiry) {
+            const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000;
+            const expiryMs = this.runtimeOptions.gcOptions?.testMode === true ?
+                0 : defaultContainerRuntimeExpiryMs;
+            const closeRuntime = () => this.closeFn({
```

Comment:
This variable is not needed. You can directly call this.closeFn() in setTimeout.